app/src/main/cpp/HandMeshRenderer.h

```diff
@@ -0,0 +1,61 @@
+/* -*- Mode: C++; tab-width: 20; indent-tabs-mode: nil; c-basic-offset: 2 -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#pragma once
+
+#include "vrb/CreationContext.h"
+
+namespace crow {
+
+enum class HandMeshStrategy { None, Skinned, Spheres };
+
+struct Controller;
+
+class HandMeshRenderer;
+typedef std::shared_ptr<HandMeshRenderer> HandMeshRendererPtr;
```

Comment:
Why a shared_ptr and not an unique_ptr? I guess we only need to have one instance of it owned by BrowserWorld

Reply:
That's right, only one instance is created, and owned only by BrowserWorld. Will fix that.